src/frequenz/sdk/config/_config_managing.py

```diff
@@ -44,7 +72,11 @@ def __init__(
         """Initialize this instance.
 
         Args:
-            config_path: The path to the TOML file with the configuration.
+            config_paths: The paths to the TOML files with the configuration. Order is
```

Comment:
maybe `Order matters as [..]` sounds nicer? :shrug:

Reply:
ChatGPT agrees with me:

> The first version sounds better, as "Order is important" is more formal and clearer than "Order matter," which is grammatically incorrect. "Order matter" should be "Order matters" to be correct, but even then, "Order is important" is a better choice for conveying the significance more clearly.

Maybe it's because copilot wrote that :laughing:

Reply:
OK, scratch that, it only preferred it because I missed the trailing "s", with the correct grammar it actually prefers your suggestion :unamused: